manifests/init.pp

```diff
@@ -278,6 +278,7 @@
   $cli_silence_ssl_warnings = false,
   $cli_username             = $st2::params::admin_username,
   $cli_password             = $st2::params::admin_password,
+  $cli_apikey               = undef,
```

Comment:
these new parameters should have docs in the section above the class

Reply:
Added the docstrings for the new params.